xc7/fasm2bels/verilog_modeling.py

```diff
@@ -780,6 +788,38 @@ def set_iostandard(self, iostandards):
 
         self.iostandard = possible_iostandards.pop()
 
+    def set_net_map(self, net_map):
+        self.net_map = net_map
+
+    def check_for_net_name(self, wire_pkey):
+        if wire_pkey in self.net_map:
+            # Top-level port names supress net names.
+            name = self.net_map[wire_pkey]
+            if name in self.top_level_signal_nets:
+                return None
+
+            # Transform net names into escaped id and bus selection:
```

Comment:
This comment should be a docstring / doctest?

Reply:
Refactored into a function and added some tests.